monai/apps/auto3dseg/bundle_gen.py

```diff
@@ -175,46 +176,71 @@ def _create_cmd(self, train_params=None):
                 # Python Fire may be confused by single-quoted WindowsPath
                 config_yaml = Path(os.path.join(config_dir, file)).as_posix()
                 base_cmd += f"'{config_yaml}'"
-
-        if "CUDA_VISIBLE_DEVICES" in params:
-            devices = params.pop("CUDA_VISIBLE_DEVICES")
-            n_devices, devices_info = len(devices), ",".join([str(x) for x in devices])
-        else:
-            n_devices, devices_info = torch.cuda.device_count(), ""
-        if n_devices > 1:
-            cmd = f"torchrun --nnodes={1:d} --nproc_per_node={n_devices:d} "
+        cmd = self.device_setting['CMD_PREFIX']
```

Comment:
The property `device_setting` is not defined.

Reply:
it's defined in self.train. self._create_cmd is only callable by self.train so device_setting should always be defined. But I can move it to init

Reply:
This device_setting should be passed by autorunner, since autorunner decides which device bundle algo train on

Reply:
I think all property used needs to be defined into the __init__ with proper annotations, or otherwise it will fail some format requirements

Reply:
changed to init